cliquet/resource/__init__.py

```diff
@@ -1046,6 +1088,17 @@ def get_parent_id(self, request):
         """
         return ''
 
+    def _extract_fields(self):
+        """Override default fields extraction from QueryString to allow
+        the permission_field to exist.
+        """
+
+        fields = super(ShareableResource, self)._extract_fields()
+        if fields:
+            fields.append(self.model.permissions_field)
+
+        return fields
```

Comment:
Well actually I hope we could ask the permission using partial responses on the collection_get endpoint.